src/Tribe/Extension.php

```diff
@@ -164,13 +164,40 @@ protected function construct() {}
 			$this->get_plugin_file(),
```

Comment:
Let's set this to a var above so it can be reused further below.

Reply:
oh, yeah, duh